### ISSUE TITLE
Sort organisation unit alphabetically in the tree

### ISSUE THREAD

**Issue by adrianq** [EyeSeeTea/organisationUnit-User-Assigment#8]
This was a bug in 2.24 and it is fixed for 2.25. Anyway probably it is relatively easy to fix

**Comment by ivanarrizabalaga** [comment]
According to Nicolay (from dhis2):

```
Hi Jose,

I just published a new version of the maintenance app for 2.24 where this should be fixed (v24.0.31).

Best regards,
```

but it looks that there is nothing changed:

https://github.com/dhis2/maintenance-app/compare/v24.0.30...v24.0.31

**Comment by adrianq** [comment]
@ivanarrizabalaga See the [commits for v24](https://github.com/dhis2/d2-ui/commits/v24). Looks like they have resolved it in [this commit](https://github.com/dhis2/d2-ui/commit/74b4109a44a5e1215249394da1cb757e595da848). Can you ported to our code?
